hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/Inventory.java

```diff
@@ -103,6 +108,30 @@
  * @since 0.0.1
  */
 public interface Inventory extends AutoCloseable, Tenants.Container<Tenants.ReadWrite> {
+    static final class InventotyUtils {
+        private static final Map<String, Class<? extends Entity<?, ?>>>entityTypes;
+        static {
+            Map<String, Class<? extends Entity<?, ?>>> m = Stream.of(
+                    Tenant.class,
+                    Environment.class,
+                    Feed.class,
+                    ResourceType.class,
+                    MetricType.class,
+                    Resource.class,
+                    Metric.class,
+                    OperationType.class,
+                    MetadataPack.class)
+            .collect(Collectors.toMap(Class::getName, Function.identity()));
```

Comment:
This is different from what the `SecurityIntegration#getClassFromName()` did. The original method looked up by lower-cased **simpleName** of the entity types, while you're using the full class name.